articlequality/feature_lists/ptwiki.py

```diff
@@ -0,0 +1,160 @@
+"""
+Portuguese Wikipedia
+++++++++++++++++++++
+"""
+import re
+
+from revscoring import Feature
+from revscoring.datasources.meta import filters, mappers
+from revscoring.features import wikitext
+from revscoring.features.meta import aggregators
+from revscoring.features.modifiers import log, max, sub
+from revscoring.languages import portuguese
+
+from . import wikipedia
+
+# Templates
+infobox_templates = wikitext.revision.template_names_matching(
+    r"Info", name="ptwiki.revision.infobox_templates")
+CN_TEMPLATES = [
+    r"Carece de fontes",
+    r"Carece de fontes2",
+    r"Carece de fontes/bloco",
+    r"Carece de fontes/bloco2"
+]
+cn_templates = wikitext.revision.template_names_matching(
+    "|".join(CN_TEMPLATES), name="ptwiki.revision.cn_templates")
+main_article_templates = wikitext.revision.template_names_matching(
+    "Main", name="ptwiki.main_article_templates")
+CITE_TEMPLATES = [
+    r"Cite",
+    r"Harvard[_ ]citation[_ ]no[_ ]brackets", r"harvnb",
+    r"Harvard citation", r"harv",
+    r"Harvard citation text", r"harvtxt",
+    r"Harvcoltxt",
+    r"Harvcol",
+    r"Harvcolnb",
+    r"Harvard citations", r"harvs",
+    r"Harvp"
+]
+cite_templates = wikitext.revision.template_names_matching(
+    "|".join(CITE_TEMPLATES), name="ptwiki.revision.cite_templates")
+SFN_TEMPLATES = [
+    r"Shortened footnote template", r"sfn",
+    r"Sfnp",
+    r"Sfnm",
+    r"Sfnmp"
+]
```

Comment:
This is good.  Looks like ptwiki uses "sfn" https://pt.wikipedia.org/wiki/Predefini%C3%A7%C3%A3o:Sfn